client/blocks/login/login-form.jsx

```diff
@@ -598,7 +598,9 @@ export class LoginForm extends Component {
 					<p className="login__form-terms">
 						{ this.props.translate(
 							// To make any changes to this copy please speak to the legal team
-							'By continuing, ' + 'you agree to our {{tosLink}}Terms of Service{{/tosLink}}.',
+							'By continuing, ' +
+								'you agree to our {{tosLink}}Terms of Service{{/tosLink}} and' +
+								' acknowledge that you have read our {{privacyLink}}Privacy Policy{{/privacyLink}}.',
```

Comment:
Let's make sure to get a confirmation from legal folks that the copy here is right - perhaps @melanieharari or @catscheer? This is in compliance with the comment above FWIW.

Reply:
@tyxla  I see that the link I initially used in the issue description wasn't correct 🤦 It's fixed now. It should have been: https://alphamattic.wordpress.com/2022/07/29/updated-branding-on-nux-pages/#comment-14399 - you can see there that the change was accepted by the legal team.

Reply:
Perfect, thanks for confirming @mcliwanow 👍

Reply:
Copy looks good. Thanks for the ping!